content/tutorial/tutorial.md

```diff
@@ -12,185 +12,189 @@ redirect_from:
   - "docs/tutorial-zh-CN.html"
 ---
 
-This tutorial doesn't assume any existing React knowledge.
+इस ट्यूटोरियल के लिए React के मौजूदा ज्ञान की कोई आवश्यकता नहीं है।
 
-## Before We Start the Tutorial {#before-we-start-the-tutorial}
+## ट्यूटोरियल शुरू करने से पहले {#before-we-start-the-tutorial}
 
-We will build a small game during this tutorial. **You might be tempted to skip it because you're not building games -- but give it a chance.** The techniques you'll learn in the tutorial are fundamental to building any React apps, and mastering it will give you a deep understanding of React.
+हम इस ट्यूटोरियल के दौरान एक छोटा गेम बनाएंगे। **आप इसे छोड़ने के लिए उत्सुक होंगे क्यूंकि आप गेम नहीं बना रहे हैं -- लेकिन इसे एक मौका दें।** ट्यूटोरियल में आपके द्वारा सीखी जाने वाली तकनीकें किसी भी React ऐप्स के निर्माण के लिए मौलिक हैं, और इसमें महारत हासिल करने से आपको React की गहरी समझ मिलेगी।
 
->Tip
+>ध्यान दें
 >
->This tutorial is designed for people who prefer to **learn by doing**. If you prefer learning concepts from the ground up, check out our [step-by-step guide](/docs/hello-world.html). You might find this tutorial and the guide complementary to each other.
+>यह ट्यूटोरियल उन लोगों के लिए बनाया गया है, जो **चीज़ें करके सीखना** पसंद करते हैं। यदि आप नीचे से ऊपर कॉन्सेप्ट्स को सीखना पसंद करते हैं, तो हमारे [कस्टेप-बाय-स्टेप गाइड](/docs/hello-world.html) देखें। आपको यह ट्यूटोरियल और गाइड एक-दूसरे के संपूरक लग सकते हैं।
```

Comment:
कस्टेप-बाय-स्टेप गाइड -> k is extra

Reply:
नीचे से ऊपर is a literal translation? can we have something better?

Reply:
`नीचे से ऊपर` best i can think of is, `yadi aap puray concepts ke sath seekhna pasand karte hai to`

Reply:
yadi aap bilkul बुनियादी concepts se seekhna pasand karte hain?